queries/dart/highlights.scm

```diff
@@ -0,0 +1,104 @@
+(dotted_identifier_list) @string
+
+; Methods
+
+(function_type
+  name: (identifier) @method)
+(super) @function
+
+; Annotations
+; TODO: highlight??
```

Comment:
Are those todos from upstream ?

Reply:
no from me

Reply:
@Akin909 seems to have interest in fine-tuning those highlights.

Reply:
Will be done in another PR I guess, let's merge that quickly to allow him to enhance Dart and maintain it.

Reply:
I will try and have a look when I get a chance although I'm pretty unfamiliar with tree sitter syntax so any link to similar previous PRs or areas of the code base would be great other wise will just try stuff and see what I get 😆

Reply:
I'd primarily just be starting with seeing if I can get methods and classes picked up and highlighted

Reply:
I would suggest to checkout the `tree-sitter-dart` repo and do a `npm install`. You can the do `tree-sitter parse <filename>` to see a parsed tree that you can correlate with the source file or follow the instructions how to run the tree-sitter web-ui locally.

In CONTRIBUTING.txt you should find an overview on the highlight categories that we have currently (of course @attribute is missing and could be added or replaced by something else). For experimenting with the syntax the tree-sitter web-ui is best: https://tree-sitter.github.io/tree-sitter/playground (but dart is missing. So you could build it locally: `tree-sitter build-wasm`